pyproject.toml

```
@@ -5,7 +5,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "sqlfluff"
-version = "3.0.2"
+version = "3.0.3"
```

```suggestion
version = "3.0.2"
```
not my problem. removing.